src/window/src/no_window.rs

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
Need to add this to lib.rs:

```rust
pub use no_window::NoWindow;

mod no_window;
```

Or else the module does not end up in the final library.

I thought we didn't need piston-input as dependency, but we need it for `NoWindow`.